tokio-macros/src/entry.rs

```diff
@@ -325,11 +325,13 @@ fn parse_knobs(
     let brace_token = input.block.brace_token;
     input.block = syn::parse2(quote_spanned! {last_stmt_end_span=>
         {
+            let body_closure = || async #body;
```

Comment:
async block is lazy, so you can omit closure.

```suggestion
            let body = async #body;
```

Reply:
til